setup.py

```diff
@@ -0,0 +1,20 @@
+from setuptools import setup
+
+setup(
+    name='scrapy-crawlera',
+    version='1.0.1',
+    license='BSD',
+    description='Crawlera middleware for Scrapy',
+    author='Scrapinghub',
+    author_email='info@scrapinghub.com',
+    url='https://github.com/scrapinghub/scrapy-crawlera',
+    py_modules=['crawlera', 'hubproxy'],
```

Comment:
Thanks @kmike , I thought hubproxy was still needed but apparently not. I'll also update the README to use `crawlera` module instead.